pkg/webhooks/server.go

```diff
@@ -29,14 +29,16 @@ type WebhookServer struct {
 	server       http.Server
 	client       *client.Client
 	policyLister v1alpha1.PolicyLister
+	filterKinds  []string
```

Comment:
Can we set the default resources in this list? For example, Nodes, Events, APIService, etc,.

Reply:
+1

Reply:
I do agree to have default resources, but instead of having them hardcoded as defaults in the code. We can pass them as default arguments to the container(update install.yaml).